app/services/foreman_ansible/override_resolver.rb

```diff
@@ -0,0 +1,18 @@
+module ForemanAnsible
+  # Service which resolves override values for hosts
+  class OverrideResolver
+    attr_reader :overrides, :ansible_variables
+
+    def initialize(host)
+      raise(Foreman::Exception, 'OverriderResolver needs a host to resolve overrides') unless host
```

Comment:
rubocop isn't happy, i think this should be
```suggestion
      raise(Foreman::Exception.new('OverriderResolver needs a host to resolve overrides')) unless host
```